Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Builder
+FROM golang:alpine as builder
+
+WORKDIR /build
+
+# Build deps
+RUN apk add --no-cache bash perl git
+
+# Copy source
+COPY . .
+
+# Build without tests due to AX.25 issues in our build container.
```

Comment:
I think this comment is misleading. The tests works fine in golang:alpine with the latest modifications in develop`